migration-engine/core/src/rpc.rs

```diff
@@ -1,5 +1,7 @@
 use crate::{CoreError, CoreResult, GenericApi};
-use jsonrpc_core::{types::error::Error as JsonRpcError, IoHandler, Params};
+#[cfg(any(feature = "sql", feature = "mongodb"))]
+use jsonrpc_core::IoHandler;
+use jsonrpc_core::{types::error::Error as JsonRpcError, Params};
```

Comment:
under this some wasm warnings.

Reply:
ugh, all these changes are here to get rid of the warnings. I give up.